org.bndtools.templating.gitrepo/src/org/bndtools/templating/jgit/GitCloneTemplate.java

```diff
@@ -127,6 +127,16 @@ public boolean accept(File path) {
         return toResourceMap(workingDir, filter);
     }
 
+    private String getRemoteBranch() {
+        if (params.branch == null) {
+            return GitCloneTemplateParams.DEFAULT_BRANCH;
+        }
+        if (params.branch.indexOf('/') == -1) {
```

Comment:
This is not right. Branches can have `/` in them. Since the local "clone" determines what it calls the remote repo, I suspect jgit is always using origin. So the test needs to be `if (!params.branch.startsWith("origin/")`.